README.md

```diff
@@ -83,7 +83,24 @@ Configure settings in the /etc/cockpit/cockpit.conf file.  See [`man cockpit.con
 
 ## Certificate setup
 
-By default, Cockpit creates a self-signed certificate for itself on first startup. This should [be customized](https://cockpit-project.org/guide/latest/https.html) for environments which use real certificates. It is recommended to use the [linux-system-roles.certificate role](https://github.com/linux-system-roles/certificate/) for that. If your machines are joined to a FreeIPA domain, or you use certmonger in a different mode already, generate a certificate for Cockpit like this:
+By default, Cockpit creates a self-signed certificate for itself on first startup. This should [be customized](https://cockpit-project.org/guide/latest/https.html) for environments which use real certificates.
+
+### Use an existing certificate
+
+If your server already has some certificate which you want Cockpit to use as well, point the `cockpit_certificate` and `cockpit_key` role options to it:
+
+```yaml
+    cockpit_certificate: /path/to/server.crt
+    cockpit_key: /path/to/server.key
```

Comment:
Please use `cockpit_cert` and `cockpit_private_key`
see https://linux-system-roles.github.io/documentation/tls_crypto_parameter_and_key_names.html for more information.

Reply:
Done, thanks!